sstables/storage.cc

```diff
@@ -114,6 +117,38 @@ future<> filesystem_storage::rename_new_file(const sstable& sst, sstring from_na
     });
 }
 
+future<> filesystem_storage::sync_cached_directory(std::string_view name) const {
+    std::string key { name };
+    auto cached_dir { _cached_dirs.find(key) };
```

Comment:
nit: similar to the above, we'd usually write this as `auto cached_dir = _cached_dirs.find(key);`

Reply:
Yep!